app/controllers/shipit/github_authentication_controller.rb

```diff
@@ -20,8 +22,9 @@ def logout
     private
 
     def sign_in_github(auth)
-      user = Shipit.github_api.user(auth[:info][:nickname])
-      User.find_or_create_from_github(user).id
+      user = User.find_or_create_from_github(auth.extra.raw_info)
```

Comment:
Oh so the output of `Shipit.github_api.user(auth[:info][:nickname])` is basically `auth.extra.raw_info`. Got it.